bucket/linkerd.json

```
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://linkerd.io/",
+    "description": "An ultralight service mesh for Kubernetes.",
+    "license": "Apache-2.0",
+    "version": "2.3.2",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/linkerd/linkerd2/releases/download/stable-2.3.2/linkerd2-cli-stable-2.3.2-windows.exe#/linkerd.exe",
+            "hash": "9f35d81cf178bb4f2babc5cb0b05a4a1775932bfb1b399214b52b6aae85a44e4"
+        }
+    },
+    "bin": "linkerd.exe",
```

I still do not see shortcut for dashboard. (https://linkerd.io/2/getting-started/)

Add something like this:

```suggestion
    "bin": "linkerd.exe",
    "shortcuts": [
        [
            "linkerd.exe",
            "Linkerd Dashboard",
            "dashboard '&'",
        ]
    ],
```

---

@Ash258 Does it make sense to add a shortcut? Linkerd is a tool very similar to istio (also in this bucket without any shortcuts). The "dashboard" command of the linkerd cli tool is not its primary function, and will only work after you set up linkerd in a kubernetes cluster (see step 2 and 3 in the getting started guide you reference). Anyways at least I think we should remove the '&' from the suggestion.

```suggestion
    "bin": "linkerd.exe",
    "shortcuts": [
        [
            "linkerd.exe",
            "Linkerd Dashboard",
            "dashboard"
        ]
    ],
```